test/test.js

```diff
@@ -172,10 +176,10 @@ test('folded headers', function(t) {
   var recp = 'bar@gmail.com', subject = 'email test why not', body = 'Why not a test email?',
   xfolded = 'This is a\r\n folded header', xfoldedtab = 'This is a\r\n\t tab folded header',
   handler = function(address, id, email) {
-    t.equal(email.headers.To,         recp);
-    t.equal(email.headers.From,       sender);
-    t.equal(email.headers.Xfolded,    xfolded.replace(/[\r\n\t]/g,''));
-    t.equal(email.headers.Xfoldedtab, xfoldedtab.replace(/[\r\n\t]/g,''));
+    t.equal(email.headers.To,                recp);
+    t.equal(email.headers.From,              sender);
+    t.equal(email.headers.get('xfolded'),    'This is a  folded header');
+    t.equal(email.headers.get('xfoldedtab'), 'This is a \t tab folded header');
```

Comment:
Why the change in the test? Why not stick with `xfolded.replace(/[\r\n\t]/g,'')` and `xfoldedtab.replace(/[\r\n\t]/g,'')`?

These actually represent what it is supposed to do.